files/system/usr/etc/chromium/vanadium_comparison.readme.md

```diff
@@ -15,77 +15,77 @@ A list of [Vanadium patches](https://github.com/GrapheneOS/Vanadium/blob/main/pa
 | [0011-Checkout-PGO-profiles](https://github.com/GrapheneOS/Vanadium/blob/main/patches/0010-Checkout-PGO-profiles.patch) | Checkout PGO profiles | No, not desired | N/A |
 | [0012-disable-checkout_nacl](https://github.com/GrapheneOS/Vanadium/blob/main/patches/0012-disable-checkout_nacl.patch) | Disables checking out nacl | Yes | [nacl is disabled in upstream Fedora](https://src.fedoraproject.org/rpms/chromium/blob/f39/f/chromium.spec#_1478) |
 | [0013-disable-first-run-welcome-page](https://github.com/GrapheneOS/Vanadium/blob/main/patches/0013-disable-first-run-welcome-page.patch) | Disable first-run welcome page | No, branding related | N/A |
```

Comment:
this file is getting deleted within the next week due to the switch to https://github.com/secureblue/hardened-chromium , so this can be dropped

the other changes are good